widget/hyperlink.go

```diff
@@ -0,0 +1,140 @@
+package widget
+
+import (
+	"image/color"
+	"net/url"
+	"os/exec"
+	"runtime"
+
+	"fyne.io/fyne"
+	"fyne.io/fyne/theme"
+)
+
+// Hyperlink widget is a text component with appropriate padding and layout.
+// When clicked, the default web browser should open with a URL
+type Hyperlink struct {
+	textProvider
+	Text      string
+	Url       string
+	Alignment fyne.TextAlign // The alignment of the Text
+	TextStyle fyne.TextStyle // The style of the hyperlink text
+}
+
+// NewHyperlink creates a new layout widget with the set text content
+func NewHyperlink(text string, sUrl string) *Hyperlink {
+	// ignore the error here, we'll continue anyway
```

Comment:
what error?

Reply:
Leftover comment, I was initially passing around a url.URL type, but realized that was work when all I really needed was a string.  To parse it out returns a possible error, and I was ignoring that error.  Since I changed it to strings, that comment is no longer needed :)

Reply:
Changed in 3fdc033528934b7e8ed483474c570a7f82cbcc9a